tools/test-proxy/Azure.Sdk.Tools.TestProxy/Record.cs

```diff
@@ -30,7 +30,7 @@ public sealed class Record : ControllerBase
         [HttpPost]
         public void Start()
         {
-            string file = RecordingHandler.GetHeader(Request, "x-recording-file");
+            string file = RecordingHandler.GetHeader(Request, "x-recording-file", true);
```

Comment:
```suggestion
            string file = RecordingHandler.GetHeader(Request, "x-recording-file", allowNulls: true);
```

NIT: Use a named parameter instead of a naked bool.